client/components/tooltip/test/index.test.tsx

```diff
@@ -82,54 +88,271 @@ describe( 'Tooltip', () => {
+		).not.toBeInTheDocument();
+		expect( screen.queryByText( 'Trigger element' ) ).toBeInTheDocument();
+		expect( handleHideMock ).not.toHaveBeenCalled();
+	} );
+
+	it( 'renders its content when opened', () => {
+		const handleHideMock = jest.fn();
+		render(
+			<ClickTooltip
+				isVisible
+				content="Tooltip content"
+				onHide={ handleHideMock }
+			>
+				<span>Trigger element</span>
+			</ClickTooltip>
+		);
+
+		jest.runAllTimers();
+
+		expect( screen.queryByText( 'Tooltip content' ) ).toBeInTheDocument();
+		expect( screen.queryByText( 'Trigger element' ) ).toBeInTheDocument();
+		expect( handleHideMock ).not.toHaveBeenCalled();
+	} );
+
+	it( 'renders and hides its content when clicked', () => {
+		const handleHideMock = jest.fn();
+		render(
+			<ClickTooltip content="Tooltip content" onHide={ handleHideMock }>
+				<span>Trigger element</span>
+			</ClickTooltip>
 		);
 
 		jest.runAllTimers();
 
 		expect(
-			screen.queryByText( 'Tooltip 1 content' )
+			screen.queryByText( 'Tooltip content' )
+		).not.toBeInTheDocument();
+
+		act( () => {
+			userEvent.click( screen.getByText( 'Trigger element' ) );
+			jest.runAllTimers();
+		} );
+
+		expect( screen.queryByText( 'Tooltip content' ) ).toBeInTheDocument();
+		expect( handleHideMock ).not.toHaveBeenCalled();
+
+		act( () => {
+			userEvent.click( screen.getByText( 'Trigger element' ) );
+			jest.runAllTimers();
+		} );
+
+		expect(
+			screen.queryByText( 'Tooltip content' )
 		).not.toBeInTheDocument();
+		expect( handleHideMock ).toHaveBeenCalled();
+	} );
+
+	it( `doesn't show or hide on hover events`, () => {
+		const handleHideMock = jest.fn();
+		render(
+			<ClickTooltip content="Tooltip content" onHide={ handleHideMock }>
+				<span>Trigger element</span>
+			</ClickTooltip>
+		);
+
 		expect(
-			screen.queryByText( 'Tooltip 2 content' )
+			screen.queryByText( 'Tooltip content' )
 		).not.toBeInTheDocument();
-		expect( handleHide1Mock ).not.toHaveBeenCalled();
-		expect( handleHide2Mock ).not.toHaveBeenCalled();
 
-		// opening the first tooltip, no need to call any hide handlers
-		act( () => userEvent.click( screen.getByText( 'Open tooltip 1' ) ) );
+		act( () => {
+			fireEvent.mouseOver( screen.getByText( 'Trigger element' ) );
+			jest.runAllTimers();
+		} );
+
+		expect(
+			screen.queryByText( 'Tooltip content' )
+		).not.toBeInTheDocument();
+		expect( handleHideMock ).not.toHaveBeenCalled();
+
+		act( () => {
+			fireEvent.mouseLeave( screen.getByText( 'Trigger element' ) );
+			jest.advanceTimersByTime( 1000 );
+		} );
 
-		expect( screen.queryByText( 'Tooltip 1 content' ) ).toBeInTheDocument();
 		expect(
-			screen.queryByText( 'Tooltip 2 content' )
+			screen.queryByText( 'Tooltip content' )
 		).not.toBeInTheDocument();
-		expect( handleHide1Mock ).not.toHaveBeenCalled();
-		expect( handleHide2Mock ).not.toHaveBeenCalled();
+		expect( handleHideMock ).not.toHaveBeenCalled();
+	} );
+} );
+
+describe( 'Tooltips', () => {
```

Comment:
I've added a new test suite to ensure the 2 variants hide each other correctly when required.